lib/const.rb

```diff
@@ -62,12 +62,13 @@
 end
 
 # Set CREW_NOT_COMPRESS from environment variable
-CREW_NOT_COMPRESS = ENV['CREW_NOT_COMPRESS']
+CREW_NOT_COMPRESS = ENV['CREW_COMPRESS']
```

Comment:
Why was this changed?  Don't we want the environment variable to match the constant?

Reply:
Corrected back to `ENV['CREW_NOT_COMPRESS']`